Administrator Guide/Managing Volumes.md

```diff
@@ -673,10 +674,10 @@ volume or only on the files which need *healing*.
     *healing*:
 
         # gluster volume heal test-volume info
-        Brick :/gfs/test-volume_0
+        Brick server1:/gfs/test-volume_0
```

Comment:
I'll fix this up when I do the next couple of pages. 

Sadly I just realized that "Setting up Volumes" is still broken with my changes. It seems that mkdocs breaks code blocks that are within a quote where as github parses them correctly, example: 

> Quote Test
 ```
 do quote things
 ```

This doesn't get fixed with my changes and is still broken on live.